todo-app/src/data/todo-data.json

```diff
@@ -0,0 +1,18 @@
+[{
+    "id":"1",
+    "task":"Get out of bed",
+    "time":"Wed Sep 13 2017",
+    "done": true
```

Comment:
In relation to my comment above: when loading your app, the checkbox for this to do is unchecked.